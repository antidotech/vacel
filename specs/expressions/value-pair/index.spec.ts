import { runSpec, parsers } from '../../__helper__'

runSpec(__dirname, {
  parser: parsers.expr,
})
