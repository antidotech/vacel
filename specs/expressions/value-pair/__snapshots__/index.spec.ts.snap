// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`value-pair basic ast: ast 1`] = `
Node {
  "base": Node {
    "loc": Object {
      "end": Object {
        "column": 6,
        "line": 1,
        "offset": 5,
      },
      "start": Object {
        "column": 1,
        "line": 1,
        "offset": 0,
      },
    },
    "name": "Cookie",
    "type": "Identifier",
  },
  "loc": Object {
    "end": Object {
      "column": 14,
      "line": 1,
      "offset": 13,
    },
    "start": Object {
      "column": 1,
      "line": 1,
      "offset": 0,
    },
  },
  "name": Node {
    "loc": Object {
      "end": Object {
        "column": 14,
        "line": 1,
        "offset": 13,
      },
      "start": Object {
        "column": 8,
        "line": 1,
        "offset": 7,
      },
    },
    "name": "Vacelin",
    "type": "Identifier",
  },
  "type": "ValuePair",
}
`;

exports[`value-pair basic format: format 1`] = `Cookie:Vacelin`;

exports[`value-pair basic token 1`] = `
Array [
  Cookie,
  :,
  Vacelin,
]
`;
